From f9ac1c9bb5692f2b8b0061b4864b1688ba432791 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Fri, 24 Apr 2020 11:00:43 +0100
Subject: [PATCH] build: Include build-db in tarball releases

SCRIPTS are not included in tarballs by default, because they might
be generated from a template. This one isn't, so use the dist_
prefix to distribute it.

Resolves: https://github.com/lathiat/avahi/issues/277
Signed-off-by: Simon McVittie <smcv@debian.org>
---
 service-type-database/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/service-type-database/Makefile.am b/service-type-database/Makefile.am
index f9fa082..3c20e0c 100644
--- a/service-type-database/Makefile.am
+++ b/service-type-database/Makefile.am
@@ -24,7 +24,7 @@ pkglibdata_DATA=
 if HAVE_PYTHON
 if HAVE_GDBM
 
-noinst_SCRIPTS=build-db
+dist_noinst_SCRIPTS=build-db
 pkglibdata_DATA+=service-types.db
 
 service-types.db: service-types
@@ -36,7 +36,7 @@ CLEANFILES = service-types.db
 endif
 if HAVE_DBM
 
-noinst_SCRIPTS=build-db
+dist_noinst_SCRIPTS=build-db
 pkglibdata_DATA+=service-types.db.pag service-types.db.dir
 
 service-types.db.pag: service-types.db
-- 
2.26.2

